requirements.txt

```diff
@@ -11,8 +11,10 @@ matplotlib
 numpy
 pandas
 pyshp
+pystac
```

Comment:
pystac-client is already listed. It includes pystac. So pystac does not need to be added explicitly.